src/ManifoldsBase.jl

```diff
@@ -268,7 +268,11 @@ distance(M::Manifold, p, q) = norm(M, p, log(M, p, q))
     embed(M::Manifold, p)
 
 Embed point `p` from the [`Manifold`](@ref) `M` into the ambient space.
-The function works only for selected embedded manifolds.
+This method is only available for manifolds where implicitly an embedding or ambient space
```

Comment:
Explicit embedding should be OK as well (through `AbstractEmbeddedManifold`).

Reply:
Similarly below.

Reply:
Yes, of course an explicit embedding is fine, too; this is the documentation of `Manifold` so I did not want to get into more specific types. For an general manifold, only the case of an implicitly assumed embedding is the explanation for this function.

Reply:
I understand now. I was a little confused because documentation for the non-mutating method on `Manifold` usually describes the entire function. It may be beneficial to point here to that section in `interface.md` for the case of an explicit embedding.

Reply:
The entire function (like for `inner`) does not cover decorators. We could of course refer from inner to the `MetricManifold` as well, but how should such a link work in REPL?

Reply:
Hm, right, REPL is not as nice here. I'd say then let's just add
```
See also: [`EmbeddedManifold`](@ref), [`project`](@ref)
```
at the end of the docstring.

Reply:
Will do so (for embed, project, but also for inner) then.

Reply:
Thanks!